src/main/app/src/hooks/useOppilaitosTyypit.ts

```diff
@@ -3,45 +3,74 @@ import _fp from 'lodash/fp';
 import {
   AMMATILLISET_OPPILAITOSTYYPIT,
   EI_TUETUT_KOULUTUSTYYPIT,
-  KORKEAKOULU_KOULUTUSTYYPIT,
+  ENTITY,
   KORKEAKOULU_OPPILAITOSTYYPIT,
-  KOULUTUSTYYPPI,
   LUKIO_OPPILAITOSTYYPIT,
+  OPPILAITOSTYYPPI_TO_KOULUTUSTYYPIT,
+  ONLY_OPH_CAN_SAVE_KOULUTUS_KOULUTUSTYYPIT,
 } from '#/src/constants';
 import { useOrganisaatioHierarkia } from '#/src/hooks/useOrganisaatioHierarkia';
 import iterateTree from '#/src/utils/iterateTree';
+import { reduce } from '#/src/utils/lodashFpUncapped';
 
 export const useOppilaitosTyypit = organisaatioOid => {
   const { hierarkia, isLoading } = useOrganisaatioHierarkia(organisaatioOid, {
     skipParents: false,
   });
 
-  const oppilaitosTyypit: Array<any> = [];
+  const oppilaitostyypit: Array<any> = [];
 
   iterateTree(hierarkia, org => {
     if (org?.oppilaitostyyppi) {
-      oppilaitosTyypit.push(org?.oppilaitostyyppi);
+      oppilaitostyypit.push(org?.oppilaitostyyppi);
     }
   });
 
   const isAmmatillinen = !_fp.isEmpty(
-    _fp.intersection(AMMATILLISET_OPPILAITOSTYYPIT, oppilaitosTyypit)
+    _fp.intersection(AMMATILLISET_OPPILAITOSTYYPIT, oppilaitostyypit)
   );
 
   const isKorkeakoulutus = !_fp.isEmpty(
-    _fp.intersection(KORKEAKOULU_OPPILAITOSTYYPIT, oppilaitosTyypit)
+    _fp.intersection(KORKEAKOULU_OPPILAITOSTYYPIT, oppilaitostyypit)
   );
 
   const isLukio = !_fp.isEmpty(
-    _fp.intersection(LUKIO_OPPILAITOSTYYPIT, oppilaitosTyypit)
+    _fp.intersection(LUKIO_OPPILAITOSTYYPIT, oppilaitostyypit)
```

Comment:
Näitä `isLukio` jne. ei tarvi enää palauttaa tästä funktiosta, koska niitä ei käytetä missään.